gym/utils/play.py

```diff
@@ -18,6 +21,55 @@
 from pygame.locals import VIDEORESIZE
 
 
+class PlayableGame:
+    def __init__(self, env: Env, keys_to_action: dict = None, zoom: float = None):
+        self.env = env
+        self.relevant_keys = self._get_relevant_keys(keys_to_action)
+        self.video_size = self._get_video_size(zoom)
+        self.screen = pygame.display.set_mode(self.video_size)
+        self.pressed_keys = []
+        self.running = True
+
+    def _get_relevant_keys(self, keys_to_action: dict) -> set:
+        if keys_to_action is None:
+            if hasattr(self.env, "get_keys_to_action"):
+                keys_to_action = self.env.get_keys_to_action()
+            elif hasattr(self.env.unwrapped, "get_keys_to_action"):
+                keys_to_action = self.env.unwrapped.get_keys_to_action()
+            else:
+                assert False, (
```

Comment:
Explicitly `raise` some `Exception` instead of a failing `assert`, you can either use one of the existing exceptions like `AttributeError` or something; or make a new one in this file to make it more descriptive.